src/pages/css/css-frameworks/css-frameworks-UIkit/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: CSS Framework UIKit
+---
+# CSS Framework UIKit
+
+UIKit is A lightweight and modular front-end framework for developing fast and powerful web interfaces.
+## Getting Started
+
+Here is a simple HTML template which includes the latest compiled and minified CSS for the UIKit library.
+
+<!DOCTYPE html>
```

Comment:
The example HTML is being processed by the browser - just add a code fence around this, with syntax highlighting: ` ```html ` and it will show the actual example HTML code.